common/localize.ts

```diff
@@ -143,6 +143,26 @@ const LOCALIZE_ID_TO_NAME = {
   // Dots and Boxes ends.
   // Gomoku starts.
   // Gomoku ends.
+  CONNECT4_GAME_NAME: {
```

Comment:
See above:

  // Connect4 starts.
  // Connect4 ends.

Put it in between them.
I gave you this comment already in 
https://github.com/yoav-zibin/shenkar/pull/22